lib/grammers-mtproto/src/mtp/encrypted.rs

```diff
@@ -25,14 +25,15 @@ const NUM_FUTURE_SALTS: i32 = 64;
 /// Used to prevent small fluctuations in the system clock.
 const SALT_USE_DELAY: i32 = 60;
 
-static UPDATE_IDS: [u32; 7] = [
+static UPDATE_IDS: [u32; 8] = [
     tl::types::UpdateShortMessage::CONSTRUCTOR_ID,
     tl::types::UpdateShortChatMessage::CONSTRUCTOR_ID,
     tl::types::UpdateShort::CONSTRUCTOR_ID,
     tl::types::UpdatesCombined::CONSTRUCTOR_ID,
     tl::types::Updates::CONSTRUCTOR_ID,
     tl::types::UpdateShortSentMessage::CONSTRUCTOR_ID,
     tl::types::messages::AffectedMessages::CONSTRUCTOR_ID,
+    tl::types::messages::InvitedUsers::CONSTRUCTOR_ID,
```

Comment:
We still need to handle this new type in https://github.com/Lonami/grammers/blob/aa8d52762d5680abbb2deeb6e639416c81d4714c/lib/grammers-mtsender/src/lib.rs#L561.